src/vconsole/vconsole-setup.c

```diff
@@ -295,6 +302,7 @@ int main(int argc, char **argv) {
         /* Let the kernel command line override /etc/vconsole.conf */
         if (detect_container() <= 0) {
                 r = parse_env_file("/proc/cmdline", WHITESPACE,
+                                   "vconsole.n.setup.vts", &n_setup_vts_s,
```

Comment:
hmm, our syntax logic here's currently very confused, but I think we should from now on stick to a syntax of <namespace>.<variable>, and thus avoid dots in the <variable> part. Hence, "vconsole.n_setup_vts" is probably a better name here, I htink.